.github/workflows/tests.yml

```
@@ -157,19 +158,13 @@ jobs:
       - name: Qodana - Code Inspection
         uses: JetBrains/qodana-action@ab695492d98902c49b33382782175107ec63ca28 # tag=v2022.2.1
         with:
-          linter: ${{ env.QODANA_LINTER }}
-          project-dir: "${{ github.workspace }}/pull_request"
-          inspected-dir: ./src/main/java # only main spoon project at first
-          results-dir:  "${{ github.workspace }}/result"
-          changes: true
-          fail-threshold: 0
+          args: --linter, ${{ env.QODANA_LINTER }}, project-dir,${{ github.workspace }}/pull_request,changes,true,fail-threshold,0
```

You could split this line on multiple lines (if GHA supports this YAML syntax, but I think it should):

```suggestion
          args: >-
              --linter,
              ${{ env.QODANA_LINTER }},
              project-dir,
              ${{ github.workspace }}/pull_request,
              changes,
              true,
              fail-threshold,
              0
```